docs/HowTo/Use-Privacy/Sign-Privacy-Marker-Transactions.md

```diff
@@ -0,0 +1,14 @@
+description: How to sign a Privacy Marker Transaction
+<!--- END of page meta data -->
+
+# Signing Private Marker Transactions
+
+Private Marker Transactions can be signed with a random key or private key. To sign Private Marker Transactions with a private key, use [`--privacy-marker-transaction-signing-key-file`](../../Reference/Pantheon-CLI/Pantheon-CLI-Syntax.md#privacy-marker-transaction-signing-key-file) when starting Pantheon.
+
+!!! important
+    In networks where gas is paid, a funded private key must be specified.
```

Comment:
This is subtle but I'd refer to it as a 'funded account'.  The address identifies an account which is accessed using the private key.  Does that make sense? Let me know if you want to discuss.

Reply:
Yep, makes perfect sense.